### PR TITLE
Scott update email/affiliation

### PR DESCRIPTION
**Pull request by scottrigby**
Update affiliation as I'm [no longer at Weaveworks](https://twitter.com/r6by/status/1615462931004817408) 🦄💖👀

### CODE REVIEW

File: MAINTAINERS [hunk 1]
```diff
@@ -7,7 +7,7 @@ In alphabetical order:
 Kingdon Barrett, Weaveworks <kingdon@weave.works> (github: @kingdonb, slack: Kingdon B)
 Daniel Holbach, Weaveworks <daniel@weave.works> (github: @dholbach, slack: dholbach)
 Stefan Prodan, Weaveworks <stefan@weave.works> (github: @stefanprodan, slack: stefanprodan)
-Scott Rigby, Weaveworks <scott@weave.works> (github: @scottrigby, slack: scottrigby)
+Scott Rigby, Weaveworks <scott@r6by.com> (github: @scottrigby, slack: scottrigby)
```
Comment (dholbach) [inline MAINTAINERS:10]: Maybe this as well?

```suggestion
Scott Rigby, Independent <scott@r6by.com> (github: @scottrigby, slack: scottrigby)
```

:people_hugging:

Comment (scottrigby) [inline MAINTAINERS:10]:
```suggestion
Scott Rigby <scott@r6by.com> (github: @scottrigby, slack: scottrigby)
```

Comment (scottrigby) [reply MAINTAINERS]: race condition 😸

**Comment by scottrigby** [comment]
Oh hey Daniel, I'll also update the rolodex. 1 sec

**Comment by dholbach** [comment]
That'll be done by the script, which sort of expects "firstname lastname, affilation <email> ..." :grimacing:

**Comment by scottrigby** [comment]
Ah right! excellent ❤️

**Comment by scottrigby** [comment]
Oh also https://github.com/fluxcd/community/blame/49debd0e896532cb696ef2aa721bc5c0aa484749/COMMUNITY.md#L101

1 sec

**Comment by dholbach** [comment]
So it'd be good to add e.g. "Independent" in there, or the script will trip over it. Sorry for the hassle!

**Review by dholbach — APPROVED** [review]

**Pull request merged**